includes/class.llms.query.php

```diff
@@ -280,6 +282,37 @@ public function maybe_404_certificate() {
 
 	}
 
+	/**
+	 * Redirect requests to old llms_my_certificate URLs to the new url.
+	 *
+	 * Redirects `/my_certificate/slug` to `/certificate/slug` maintaining
+	 * translations.
+	 *
+	 * This will only redirect if `$wp_query` detects a 404 and a certificate
+	 * exists with the parsed slug. This check is important to prevent against
+	 * collisions which are theoretically possible, though probably unlikely.
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public function maybe_redirect_certificate() {
+
+		global $wp, $wp_query;
+
+		$url = home_url( $wp->request );
+		$old = sprintf( '/%s/', _x( 'my_certificate', 'slug', 'lifterlms' ) );
+		if ( $wp_query->is_404() && str_contains( $url, $old ) ) {
```

Comment:
`str_contains()` requires PHP 8+
```suggestion
		if ( $wp_query->is_404() && false !== strpos( $url, $old ) ) {
```